db/schema.rb

```diff
@@ -56,6 +69,7 @@
     t.integer "price"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.integer "service_duration"
     t.index ["user_id"], name: "index_kondos_on_user_id"
   end
 
```

Comment:
Won't really matter but I learned recently that there is no need for us to submit changes on this file. The reason being is that this file is fully dependent on the migrations so running `rails db:migrate` should generate the right `schema.rb` depending on how updated the repo's migrate folder.